src/databricks/labs/ucx/install.py

```diff
@@ -860,6 +861,19 @@ def _remove_install_folder(self):
         except InvalidParameterValue:
             logger.error("Error deleting install folder")
 
+    def validate_step(self, step: str) -> bool:
+        job_id = self._state.jobs[step]
+        logger.debug(f"Validating {step} workflow: {self._ws.config.host}#job/{job_id}")
+        completed_runs = list(self._ws.jobs.list_runs(completed_only=True, job_id=job_id))
```

Comment:
what if the assessment run is in progress? please add a test case as well

Reply:
Done